test/Frame.spec.jsx

```diff
@@ -313,6 +313,63 @@ describe('The Frame Component', () => {
     );
   });
 
+  it('should call contentWillUnmount on unmount', done => {
+    div = document.body.appendChild(document.createElement('div'));
+    const willUnmount = sinon.spy();
+
+    function Parent() {
+      const [isClosed, setClosed] = React.useState(false);
+
+      useEffect(() => {
+        setTimeout(() => {
+          setClosed(true);
+        }, 1500);
+      }, []);
+
+      if (!isClosed) {
+        return (
+          <Frame
+            contentWillUnmount={() => {
+              willUnmount();
+              done();
+            }}
+          />
+        );
+      }
+      return null;
+    }
+    ReactDOM.render(<Parent />, div);
+  });
+
+  it('should not error when null is passed in contentWillUnmount', () => {
```

Comment:
same as above should error if passed null as prop types expects a `func`

Reply:
Can you check the test of "should error"?